lib/maintainers.nix

```diff
@@ -287,6 +287,7 @@
   joelmo = "Joel Moberg <joel.moberg@gmail.com>";
   joelteon = "Joel Taylor <me@joelt.io>";
   johbo = "Johannes Bornhold <johannes@bornhold.name>";
+  johnmh = "John M. Harris, Jr. <johnmh@openblox.org>";
```

Comment:
The attribute name here should be your GitHub handle as it says at the top of the file.

Reply:
@vyp That's absolutely horrible. I shouldn't even have to have a GitHub handle.

Reply:
It has to be unique like this otherwise what if https://github.com/johnmh ever wants to maintain anything. Also it makes it easier to @ ping people.

I agree with the sentiment that needing a GitHub account is not great, but seeing as though to add a package to the official nixpkgs repository, you have to make a GitHub "pull request" adding it, which requires one to sign up to and use GitHub, I don't see *that much* of an issue here in requiring it. Might as well utilize GitHub handles if nixpkgs is hosted on GitHub in my opinion.

Reply:
I don't actually log into the GitHub website unless I actually need to make a 'pull request'. I go by 'JohnMH' on everything but GitHub, IRC included. "pinging" me here wouldn't help anyone.

Reply:
That's not ideal, but acceptable. Actually `maintainers.nix` already contains some attributes that are different from GitHub accounts of their owners _and_ conflict with names of accounts of unrelated people who luckily do not contribute to Nix. (I don't remember who exactly.)

Reply:
In your GitHub settings under notifications you can set it so that you get an email if you get @ mentioned. And even so, that still means that https://github.com/johnmh would be forced into using something *other* than their username, and that would just makes things worse. The best and easiest way is to stick to exactly what your GitHub username is.

Reply:
@orivej, I know that @edolstra is one. 😝